### PR TITLE
Add helptexts for berry bushes

### PR DESCRIPTION
**Pull request by gunchleoc**
@Noordfrees Can you write the helptexts for the pond? You know the tribe best.

### CODE REVIEW

File: data/tribes/initialization/frisians/units.lua [hunk 1]
```diff
@@ -894,183 +894,253 @@ tribes:new_tribe {
...
          helptexts = {
+            -- TRANSLATORS: Helptext for a frisian immovable: One of the berry bushes
+            purpose = _("This berry bush is ready for harvesting.")
          }
       },
       {
          name = "berry_bush_juniper_tiny",
          helptexts = {
+            -- TRANSLATORS: Helptext for a frisian immovable: One of the berry bushes
+            purpose = _("This berry bush has just been planted.")
```
Comment (Noordfrees) [inline data/tribes/initialization/frisians/units.lua:954]: Juniper is not a berry, please use just "This bush …" for this species

**Comment by Noordfrees** [comment]
Suggestions:

**pond_dry**
When claydiggers dig up earth, they leave holes in the ground. These holes vanish after a while. Aqua farms can use them as ponds to grow fish in them, whereas charcoal burners erect their charcoal stacks in them.

**pond_growing**
Fish are growing in this pond. A fisher working from an aqua farm will be able to catch them when they are bigger.

**pond_mature**
Fish are living in this pond. A fisher working from an aqua farm can catch them as food.

**pond_burning**
A charcoal stack is burning in this earthen hole. When it has burnt down, a charcoal burner will be able to gather coal from it.

**pond_coal**
A charcoal stack, which had been erected in this earthen hole, is ready for a charcoal burner to gather coal from it.

File: data/tribes/initialization/frisians/units.lua (hunk 1 above)
Comment (gunchleoc) [reply data/tribes/initialization/frisians/units.lua:954]: Actually, juniper is not a bush, it's a shrub. And then we have to explain that the fruit collector collects fruit from bushes and shrubs, and not from "berry bushes" etc. I'd say let's simply use game terminology here.

File: data/tribes/initialization/frisians/units.lua [hunk 2]
```diff
@@ -894,183 +894,253 @@ tribes:new_tribe {
...
          helptexts = {
+            -- TRANSLATORS: Helptext for a frisian immovable: One of the berry bushes
+            purpose = _("This berry bush is growing.")
          }
       },
       {
          name = "berry_bush_blueberry_medium",
          helptexts = {
+            -- TRANSLATORS: Helptext for a frisian immovable: One of the berry bushes
+            purpose = _("This berry bush is growing.")
```
Comment (hessenfarmer) [inline data/tribes/initialization/frisians/units.lua]: I just remembered myself of medium bushes being flowering to produce honey. So I suggest to change the helptexts of medium bushes to:
This berry bush is flowering. Honey can be produced from it by a beekeeper.

**Review by hessenfarmer — CHANGES_REQUESTED** [review]

File: data/tribes/initialization/frisians/units.lua (hunk 2 above)
Comment (gunchleoc) [reply data/tribes/initialization/frisians/units.lua]: Good idea. I have added this to the medium fields etc. too.

**Review by hessenfarmer — APPROVED** [review]
LGTM now

**Pull request merged**